src/main/java/com/ea/restaurant/grpc/interceptors/GrpcServerInterceptor.java

```diff
@@ -0,0 +1,25 @@
+package com.ea.restaurant.grpc.interceptors;
+
+import io.grpc.Context;
+import io.grpc.Contexts;
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+import net.devh.boot.grpc.server.interceptor.GrpcGlobalServerInterceptor;
+
+@GrpcGlobalServerInterceptor
+public class GrpcServerInterceptor implements ServerInterceptor {
+  public static final Context.Key<Object> CONTEXT_METADATA = Context.key("context_metadata");
```

Comment:
Use enums or constant static strings in order to get the key name

Reply:
Done.